todo_app/flask_config.py

```diff
@@ -7,3 +7,6 @@ def __init__(self):
         self.SECRET_KEY = os.environ.get('SECRET_KEY')
         if not self.SECRET_KEY:
             raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
+
+        self.LOG_LEVEL = os.environ.get('LOG_LEVEL')
```

Comment:
It might be good to specify a default value here, since currently your program errors if this isn't set but we could sensibly pick a standard. For example:
```suggestion
        self.LOG_LEVEL = os.environ.get('LOG_LEVEL', 'INFO')
```